### PR TITLE
Minor edits.

### PR DESCRIPTION
**Pull request by rmcd1024**
Most substantive changes:

* removed references to ggplot2 (to facilitate using this lesson earlier in the sequence)
* revised the treatment of challenge 4 to focus on vectorization

### CODE REVIEW

File: 09-vectorisation.Rmd [hunk 1]
```diff
@@ -61,47 +61,61 @@ y:  6  7  8  9
...
-> Refresh your ggplot skills by plotting population in millions against year.
+> On a single graph, plot population, in
+> millions, against year, for all countries. Don't worry about
+>identifying which country is which.
 >
+> Repeat the exercise, graphing only for China, India, and
+>Indonesia. Again, don't worry about which is which.
+>
+
+Comparison operators, logical operators, and many function are also
```
Comment (naupaka) [inline 09-vectorisation.Rmd]: function -> functions

**Comment by naupaka** [comment]
Happy to merge after you fix the typo. Very nice!

The other thing I thought of as I went through the changes was your use of a function for the solution to Challenge 4. That would also necessitate this going later in the sequence (after the lesson on functions). Any reason to not just have the solution without a function? Or present both non-function and function solutions?

**Comment by rmcd1024** [comment]
I can delete the use of the function in Challenge 4. Let me know.

**Comment by naupaka** [comment]
Nope, this is good. I think it's nice to leave in, but I appreciate your addition of a non-functional (hah!) solution.

**Pull request merged**